packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -916,6 +936,47 @@ void WebView::HandleMethodCall(
     result->Success(flutter::EncodableValue(webview_instance_->GetScrollX()));
   } else if (method_name.compare("getScrollY") == 0) {
     result->Success(flutter::EncodableValue(webview_instance_->GetScrollY()));
+  } else if (method_name.compare("loadFlutterAsset") == 0) {
+    if (std::holds_alternative<std::string>(arguments)) {
+      std::string key = std::get<std::string>(arguments);
+      std::string path;
+      char* resPath = app_get_resource_path();
+      if (resPath) {
+        path = std::string("file://") + resPath + "flutter_assets/" + key;
+        free(resPath);
+        webview_instance_->LoadURL(path);
+        result->Success();
+        return;
+      }
+    }
+    result->Error("InvalidArguments", "Please set 'key' properly");
+  } else if (method_name.compare("loadHtmlString") == 0) {
+    std::string html;
+    std::string baseUrl;
+    if (!GetValueFromEncodableMap(arguments, "html", &html)) {
+      result->Error("InvalidArguments", "Please set 'html' properly");
+      return;
+    }
+    if (GetValueFromEncodableMap(arguments, "baseUrl", &baseUrl)) {
```

Comment:
`baseUrl` is not used.

Reply:
You're right. lwe is currently unable to process it, so it only shows the log.